introduction.md

```diff
@@ -6,33 +6,44 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB **software** integrates different home automation systems, devices and technologies into a single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+## Getting Started
 
-## openHAB Structure
-
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+Now that you know what openHAB is, you may be wondering how to get started.
+We highly recommend that you read the next article titled [Concepts]({{base}}/concepts/index.html).
+This article introduces a number of important ideas that will help you as you install and begin to configure openHAB for the first time.
+openHAB runs on many popular platforms including Linux, Windows and Mac OSx.
+You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/intex.html) article.
```

Comment:
The url is wrong.

Should be: `{{base}}/installation/index.html`

Reply:
Thank you - will be fixed in my next commit.